marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -111,4 +112,33 @@ describe('TilesView component', () => {
         expect(getByText('testFlow')).toBeInTheDocument();
     });
 
+    test('Verify Load tile displays from toolbar', async () => {
+        const {getByLabelText, getByText, queryByText} = render(<AuthoritiesContext.Provider value={mockDevRolesService}><TilesView/></AuthoritiesContext.Provider>);
+
+        // Load tile not shown initially
+        expect(queryByText("icon-load")).not.toBeInTheDocument();
+        expect(queryByText("title-load")).not.toBeInTheDocument();
+
+        await axiosMock.get['mockImplementation']((url) => {
+            switch (url) {
+                case '/api/flows':
+                    return Promise.resolve(data.flows)
+                case '/api/steps/ingestion':
+                    return Promise.resolve(data.loads)
+                default:
+                    return Promise.reject(new Error('not found'))
+            }
+        })
+        fireEvent.click(getByLabelText("tool-load"));
+
+        // Load tile shown after click
+        expect(await(waitForElement(() => getByLabelText("icon-load")))).toBeInTheDocument();
+        expect(getByLabelText("title-load")).toBeInTheDocument();
+        // Default list view
+        expect(getByLabelText("switch-view")).toBeInTheDocument();
+        expect(getByLabelText("switch-view-card")).toBeInTheDocument();
+        expect(getByLabelText("switch-view-list")).toBeInTheDocument();
+        expect(getByLabelText("add-new-list")).toBeInTheDocument();
```

Comment:
Can we add some assertions for card view. Similar to what Ryan has in his PR, only this one is being tested as a developer user, to ensure test plan coverage?